source/clear-linux/guides/maintenance/mixin.rst

```diff
@@ -104,64 +119,78 @@ packages, run the following to create your local mix.
      and multiplying it by 1000.  For example, if your upstream version is
      21530, your custom version will be 21530000.  For each subsequent call
      to mixin, mixer will increment the version by 10.  For example,
-     21530010, 21530020, etc. 
+     21530010, 21530020, etc.
 
-Migrate your Clear Linux system to your custom mix
-**************************************************
+.. _migrate-mix:
+
+Migrate to your custom mix
+==========================
 
 Before you can use your custom bundle, you must migrate your |CL| system
-to your custom mix to make the bundle accessible.
+to your custom mix to make the bundle accessible:
+
+.. code-block:: bash
 
-.. code-block:: console
+   sudo swupd update --migrate
 
-   $ sudo swupd update --migrate
+After you migrate, the version of your system will be your last custom
+version number as noted in the previous section.
 
-After you migrate, the version of your |CL| system switches over to your
-last custom version number as noted in the previous section. 
+You can continue to create new bundles with :command:`mixin` while you are in
+your custom version of |CL|.  You do not need to migrate again. However each
+time you create new bundles with mixin, you must run :command:`swupd update`
+to make the new bundles visible.
 
-You can continue to create new bundles with :command:`mixin` 
-while you are in your custom version of |CL|.  You do not need to migrate
-again. However, you must run :command:`swupd update` again to update your
-system in order to make those bundles visible. 
+.. _add-bundle-to-system:
 
-Add your custom bundle to your system
-*************************************
+Add custom bundle to your system
+================================
 
-#. Get a listing of your newly-created bundle.
+Get a listing of your newly-created bundle:
 
-   .. code-block:: console
+.. code-block:: bash
 
-      $ sudo swupd bundle-list -a
+   sudo swupd bundle-list -a
 
    The listing includes all upstream bundles.
```

Comment:
Please left-align  Ln 155 so it doesn't appear as part of the command above (unintended I know).

Reply:
Revision looks good.